### ISSUE TITLE
Doubt about SSL comparison

### ISSUE THREAD

**Issue by jesusmolrdv** [deep-optimization/ScaleKD#3]
First of all, thank you for sharing the code for this very interesting paper!

While reading your arXiv preprint, I noticed the comparison with "Self-supervised pre-training" in Table 4, where you mention "⇒ denotes transfer learning".

I would like to clarify the setup for the models being compared in this section. Specifically, are you comparing:

1. A model that was first pre-trained using iBot on IN22k (in a self-supervised manner) and then fine-tuned in a supervised way on IN1K?
2. A model that was distilled from BEiT-L/14 using the loss function described in Equation 5 of your paper? If this is the case, could you please provide details on how the BEiT-L/14 model was originally trained?

Additionally, for the distillation process mentioned in point 2, are you using the full ScaleKD loss (including the logits signal from the teacher) or a variant without the logits signal?

Thank you in advance for your clarification!

**Comment by jwfanDL** [comment]
I appreciate your interest in our paper. 

1. Yes, the models in self-supervised pre-training results in Table 4 are all the official results in the original paper, where the models are self-supervised pre-trained on ImageNet-22K and then supervised fine-tuned on ImageNet-1K.
2. As we described in the caption of Table 3, the BEiT-L is pre-trained by EVA. If you want to know details about this pre-training, I recommend you check the details in the original paper. Also you can obtain the checkpoints on both the official project and the mmpretrain project. 

The full ScaleKD loss contains the classical KD loss rather than its variants. 

I hope these answers could help you.

**Comment by jesusmolrdv** [comment]
Yes, this answers my questions. Thanks!
